src/ceph.in

```diff
@@ -254,6 +254,14 @@ GLOBAL_ARGS = {
 
 
 def parse_cmdargs(args=None, target=''):
+    """
+    Consume generic arguments from the start of the ``args``
+    list.  Call this first to handle arguments that are not
+    part of a server-side command's parameters.
```

Comment:
I know it's hard to describe this, but...the 'generic args' are server-side too.  hm.  Maybe just call them "generic args that are not part of the 'argdesc' parameters" as a signpost?

Reply:
I'll update this comment